src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+@import 'variables';
+
+.card {
+  background-color: $white;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $lightGray;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    display: block;
+    padding: 32px 19px 40px;
+  }
+
+  &__name {
+    font-size: $baseFontSize;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $darkBlue;
+    padding: 0 16px 4px;
```

Comment:
Also add the padding for the card and remove the horizontal padding everywhere

Reply:
+++